lib/src/Pos/Launcher/Configuration.hs

```diff
@@ -107,6 +109,28 @@ instance FromJSON Configuration where
 
             -- else default to RequiresMagic
             | otherwise -> pure RequiresMagic
+        ccTxValRules <- if
+            | HM.member "txValidationRules" o -> do
+                valRulesObj <- o .: "txValidationRules"
+
+                attribResEpoch <- valRulesObj .: "attribResrictEpoch"
+                aaSize <- valRulesObj .: "addrAttribSize"
+                taSize <- valRulesObj .: "txAttribSize"
+                -- attribResEpoch is used twice because
+                -- the second field of TxValidationRules
+                -- is for the current Epoch, which is taken
+                -- from the state later on.
+                pure $ TxValidationRules
+                           (EpochOrSlot . Left $ EpochIndex attribResEpoch)
+                           (EpochOrSlot . Left $ EpochIndex attribResEpoch)
+                           aaSize
+                           taSize
+            -- Defaults to this value in order for golden tests to pass
+            | otherwise -> pure $ TxValidationRules
+                                      (EpochOrSlot . Left $ EpochIndex 0)
+                                      (EpochOrSlot . Left $ EpochIndex 0)
+                                      99999999
+                                      99999999
```

Comment:
So this is the test that would fail if we didn't have a default JSON value: https://github.com/input-output-hk/cardano-sl/blob/ac792f775526ec9ed51cf83143c69253fc158860/lib/test/Test/Pos/Launcher/Json.hs#L44

I could rewrite the goldenfile to include a `TxValidationRules`.